ts/navigation/AuthenticatedStackNavigator.tsx

```diff
@@ -225,7 +226,17 @@ const AuthenticatedStackNavigator = () => {
         name={WalletPaymentRoutes.WALLET_PAYMENT_MAIN}
         component={WalletPaymentNavigator}
         options={{
-          /* FIXME: Using react-navigation 6.x we can achive this using a Stack.Group inside the main payment navigator */
+          gestureEnabled: isGestureEnabled
+        }}
+      />
+      {/* 
+            This screen is outside the main payment navigator to enable the slide from bottom animation.
+            FIXME: Using react-navigation 6.x we can achive this using a Stack.Group inside the main payment navigator
+      */}
```

Comment:
Same as above

Reply:
Added in https://github.com/pagopa/io-app/pull/5130/commits/c43a57b571feaa7222f66b7b8861f2d28b01f3d9